Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build the manager binary
-FROM golang:1.20.5 as builder
+FROM --platform=${BUILDPLATFORM} golang:1.19 as builder
```

Comment:
This actually fixed the issue for multi-platform build which was broken.

Reply:
--platform=${BUILDPLATFORM} makes sense but we want 1.20 or 1.21 goLang version

Reply:
@oscarr-ntap will change to the 1.21.